rust/Cargo.toml

```diff
@@ -45,11 +44,14 @@ crate-type = ["staticlib"]
 [profile.release]
 lto = true
 
+# Revision corresponds to the partially-complete auto-shielding version, with the NU5
+# consensus branch ID and activation heights backported onto it.
+# https://github.com/zcash/librustzcash/pull/555
 [patch.crates-io]
-zcash_client_backend = {git = "https://github.com/zcash/librustzcash", branch = "autoshielding-poc-sdk-dnm"}
-zcash_client_sqlite = {git = "https://github.com/zcash/librustzcash", branch = "autoshielding-poc-sdk-dnm"}
-zcash_primitives = {git = "https://github.com/zcash/librustzcash", branch = "autoshielding-poc-sdk-dnm"}
-zcash_proofs = {git = "https://github.com/zcash/librustzcash", branch = "autoshielding-poc-sdk-dnm"}
+zcash_client_backend = { git = 'https://github.com/zcash/librustzcash.git', rev='098dcc2ae02841e6a2b17bbb6253de970b1ff41c' }
+zcash_client_sqlite = { git = 'https://github.com/zcash/librustzcash.git', rev='098dcc2ae02841e6a2b17bbb6253de970b1ff41c' }
```

Comment:
I see there's a PR on the Android SDK that actually points to another revision `36dad6282ef8101a147512ef6daa34bbb2a4b55a`

see
[sdk-lib/Cargo.toml](https://github.com/zcash/zcash-android-wallet-sdk/pull/532/files#diff-bf8259e943f625b203c40d83878da32e98c5df29e9fedb7d27ce8052e0220df2)